lib/mollie/settlement.rb

```diff
@@ -2,12 +2,14 @@ module Mollie
   class Settlement < Base
     attr_accessor :id,
                   :reference,
-                  :settled_datetime,
+                  :created_at,
```

Comment:
The API returns a set of links which allow you to directly retrieve `payments, `refunds`, and `chargebacks` for a certain settlement.

Can we add helper methods for these `links` that allow users to directly retrieve these collections?Would be nice if we could use `settlement.payments` to retrieve a list of payments associated with this settlement.